src/EnergyPlus/Fans.cc

```diff
@@ -1469,6 +1470,7 @@ namespace Fans {
             ReportSizingOutput(Fan(FanNum).FanType, Fan(FanNum).FanName, "Design Motor Efficiency []", Fan(FanNum).MotEff);
             ReportSizingOutput(Fan(FanNum).FanType, Fan(FanNum).FanName, "Design VFD Efficiency []", Fan(FanNum).VFDEff);
             ReportSizingOutput(Fan(FanNum).FanType, Fan(FanNum).FanName, "Design Combined Efficiency []", Fan(FanNum).FanEff);
+            ReportSizingOutput(Fan(FanNum).FanType, Fan(FanNum).FanName, "Design Point Fan Energy Index", Fan(FanNum).DesignPointFEI);
```

Comment:
So why is this specific eio report different from other reports that are reported to both the eio file and the table reports? Autosized air flow or capacity for example. What if a GUI is reading the eio file instead of the html reports?

Reply:
This is not a sizing result. It's a rating point, like the DX coil SEER.

Reply:
Well, there is a lot of non-sizing information in the eio SEER being one of them. I look at the eio as a place where information is summarized. At one point I thought it was only autosized information reported to the eio but realized other info was reported to the eio. I don't care either way, I just want to know what information goes where.

    ! <DX Cooling Coil Standard Rating Information>, Component Type, Component Name, Standard Rating (Net) Cooling Capacity {W}, Standard Rated Net COP {W/W}, EER {Btu/W-h}, SEER {Btu/W-h}, IEER {Btu/W-h}
     DX Cooling Coil Standard Rating Information, Coil:Cooling:DX:SingleSpeed, SYS 1 FURNACE DX COOL SNGLSPD COOLING COIL, 7100.3, 2.65, 9.03, 9.59, 9.45

Reply:
But I don't think fans have a section in the eio, do they?  And the problem here is that it's being reported as "Component Sizing Information" which isn't right. I suppose we could add a fans section to the eio, but that's probably out of scope for this.

Reply:
No, the fans don't have a section for summarizing information. I also noticed that this FEI report to the eio is only for component models and line 1511 implies the FEI is only for non-component models. So for your reasoning and this I agree, delete the line (1473) that reports to the eio.